src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/helper/builders/OffenderProgramProfileBuilder.kt

```diff
@@ -29,6 +31,36 @@ class OffenderProgramProfileBuilder(
       payBands.addAll(this@OffenderProgramProfileBuilder.payBands.map { it.build(this) })
     }
 
+  fun build(offenderBooking: OffenderBooking): OffenderProgramProfile =
+    OffenderProgramProfile(
```

Comment:
Interesting that this builder now has 2 build methods depending upon whether its parent is an offender booking or a course allocation. This means there is some duplication so we might conslidate this later 🤔

Reply:
we have a similar issue around adjudication parties